.github/workflows/self-hosted-kind.yml

```diff
@@ -0,0 +1,48 @@
+name: vHive self-hosted KinD
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  vhive-self-hosted-kind:
+    name: vHive self-hosted KinD
+    runs-on: [stock-knative]
```

Comment:
hmm, I thought you need to specify the "self-hosted" label as well.

Reply:
Along with "self-hosted", a custom lablel like [this](https://github.com/ease-lab/vhive/blob/fffc98ca29d6afef9dd696e746c1f29ccb20a0b5/.github/workflows/cri_test.yml#L31)

Reply:
The label `stock-knative` is applied to self-hosted runners only anyway so it seems redundant to me.